clients/web/src/views/layout/HeaderUserView.js

```diff
@@ -13,10 +13,9 @@ girder.views.LayoutHeaderUserView = girder.View.extend({
         },
 
         'click a.g-logout': function () {
-            girder.restRequest({
-                path: 'user/authentication',
-                type: 'DELETE'
-            }).done(_.bind(function () {
+            girder.logout();
+
+            girder.events.once('g:logout.success', _.bind(function () {
```

Comment:
Ah, I see the duplicated triggering of ``g:login`` but that seems like an orthogonal issue.  I'll move the ``girder.currentUser`` stuff to the ``login()`` and ``logout()`` functions.